plugin/storage/es/spanstore/writer.go

```diff
@@ -131,7 +131,7 @@ func (s *SpanWriter) WriteSpan(span *model.Span) error {
 }
 
 func indexNames(span *model.Span) (string, string) {
```

Comment:
something I don't understand. The issue said there is a discrepancy between reader and writer in the use of UTC, but this function is global. Is the function shared by the reader/writer? If not, I think that's the missed opportunity to remove the discrepancies.

I see that the reader has another function `func indexWithDate(prefix string, date time.Time) string {`.

So it looks like they both re-implement similar functionality instead of relying on a shared struct doing it for them.

Reply:
agree I will address it in a separate PR